src/Promitor.Core/Extensions/AzureCloudExtensions.cs

```diff
@@ -27,5 +28,22 @@ public static AzureEnvironment GetAzureEnvironment(this AzureCloud azureCloud)
                     throw new ArgumentOutOfRangeException(nameof(azureCloud), "No Azure environment is known for");
             }
         }
+
+        public static Uri GetAzureAuthorityHost(this AzureCloud azureCloud)
```

Comment:
Good thinking! Would you mind adding a unit test to ensure that it returns the correct values please?

Reply:
If not, happy to do a PR after this one - that's ok

Reply:
I updated the unit tests 😄